src/components/loading/loading.js

```diff
@@ -89,10 +89,10 @@ class Loading extends BaseComponent {
     }
 
     this.player.addEventListener(this.player.Event.PLAYER_STATE_CHANGED, e => {
-      if (e.payload.newState.type === 'idle' || e.payload.newState.type === 'playing' || e.payload.newState.type === 'paused') {
+      if ((e.payload.newState.type === 'idle' || e.payload.newState.type === 'playing' || e.payload.newState.type === 'paused') && this.state.afterFirstPlay) {
         this.props.updateLoadingSpinnerState(false);
       }
-      else {
+      else if (this.state.afterFirstPlay) {
```

Comment:
Why we need this `if` statement?

Reply:
Because if we won't have it - we will have a spinner after first play
I want to ignore all player_state_changed events before first play

Reply:
so first check this state, don't use it twice
`this.player.addEventListener(this.player.Event.PLAYER_STATE_CHANGED, e => {`
 `if (!this.state.afterFirstPlay) return;`

and now u can give up both